docs/reference/cat.asciidoc

```diff
@@ -113,13 +113,13 @@ by all the shards, not number of documents).  The `/_cat/indices` API
 is ideal.  You only need to add three things to the API request:
 
 . The `bytes` query string parameter with a value of `b` to get byte-level resolution.
-. The `s` (sort) parameter with a value of `store.size:desc` to sort the output
-by shard storage in descending order.
+. The `s` (sort) parameter with a value of `index:desc` to sort the output
+by index name in descending order.
```

Comment:
This will fix the flaky test but it doesn't really make sense in the context of example - we want to find largest index here so sorting on index name gives us nothing. Maybe add `index:desc` (or rather `index:asc`) in addition to `store.size`, not instead?

Reply:
Good catch, I didn't look closely enough at the surrounding context. I've pushed a change that uses both the shard size and index name for sorting.